app/Http/Controllers/StudentController.php

```diff
@@ -15,7 +19,17 @@ class StudentController extends Controller
      */
     public function index()
     {
-        echo "from index";
+        if(!Schema::hasTable("students")) {
+            throw new \Exception("Trazena tabela ne postoji u bazi.");
+        }
+
+        $students = Student::all();
+        if ($students->isEmpty()) {
+            throw new \Exception("Tabela je prazna.");
```

Comment:
No need to throw if empty, but rather change the blade template and draw the else case (I remember that you check are the students set)